- Opera can also do DOM, so include opera into list of browsers to whom display
  toggle visibilty links on issue listing page.

- also Safari was left out in few similiar places.

- also includes dynCalendar fix for opera

--- ./templates/en/support_emails.tpl.html~	2005-03-14 17:03:12.380217651 +0200
--- ./templates/en/support_emails.tpl.html	2005-03-14 17:03:13.522029647 +0200
@@ -60,7 +60,7 @@
             {if $current_role >= $roles.standard_user}
             [ <a href="mail_queue.php?iss_id={$smarty.get.id}" class="link" title="view the history of sent emails">Mail Queue Log</a> ]
             {/if}
-            {if $browser.ie5up or $browser.ns6up or $browser.gecko or $browser.safari}
+            {if $browser.ie5up or $browser.ns6up or $browser.gecko or $browser.safari or $browser.opera5up}
             [ <a id="support_emails_link" class="link" href="javascript:void(null);" onClick="javascript:toggleVisibility('support_emails');">{get_innerhtml element_name="support_emails" total=$emails|@count}</a> ]
             {/if}
           </td>
--- ./templates/en/email_drafts.tpl.html~	2005-03-14 16:10:41.000000000 +0200
--- ./templates/en/email_drafts.tpl.html	2005-03-14 17:04:17.298528596 +0200
@@ -44,7 +44,7 @@
             <b>Drafts ({$drafts|@count})</b>
           </td>
           <td align="right" class="default">
-            {if $browser.ie5up or $browser.ns6up or $browser.gecko or $browser.safari}
+            {if $browser.ie5up or $browser.ns6up or $browser.gecko or $browser.safari or $browser.opera5up}
             [ <a id="drafts_link" class="link" href="javascript:void(null);" onClick="javascript:toggleVisibility('drafts');">{get_innerhtml element_name="drafts" total=$drafts|@count}</a> ]
             {/if}
           </td>
--- ./templates/en/notes.tpl.html~	2005-03-14 16:10:41.000000000 +0200
--- ./templates/en/notes.tpl.html	2005-03-14 17:04:31.667162750 +0200
@@ -54,7 +54,7 @@
             <b>Internal Notes ({$notes|@count})</b>
           </td>
           <td align="right" class="default">
-            {if $browser.ie5up or $browser.ns6up or $browser.gecko or $browser.safari}
+            {if $browser.ie5up or $browser.ns6up or $browser.gecko or $browser.safari or $browser.opera5up}
             [ <a id="notes_link" class="link" href="javascript:void(null);" onClick="javascript:toggleVisibility('notes');">{get_innerhtml element_name="notes" total=$notes|@count}</a> ]
             {/if}
           </td>
--- ./templates/en/phone_support.tpl.html~	2005-03-14 16:10:41.000000000 +0200
--- ./templates/en/phone_support.tpl.html	2005-03-14 17:04:39.095939576 +0200
@@ -34,7 +34,7 @@
           <b>Phone Calls ({$phone_entries|@count})</b>
         </td>
         <td align="right" class="default">
-            {if $browser.ie5up or $browser.ns6up or $browser.gecko or $browser.safari}
+            {if $browser.ie5up or $browser.ns6up or $browser.gecko or $browser.safari or $browser.opera5up}
             [ <a id="phone_support_link" class="link" href="javascript:void(null);" onClick="javascript:toggleVisibility('phone_support');">{get_innerhtml element_name="phone_support" total=$phone_entries|@count}</a> ]
             {/if}
         </td>
--- ./templates/en/time_tracking.tpl.html~	2005-03-14 16:10:41.000000000 +0200
--- ./templates/en/time_tracking.tpl.html	2005-03-14 17:04:25.572166312 +0200
@@ -35,7 +35,7 @@
             <b>Time Tracking ({$time_entries|@count})</b>
           </td>
           <td align="right" class="default">
-            {if $browser.ie5up or $browser.ns6up or $browser.gecko or $browser.safari}
+            {if $browser.ie5up or $browser.ns6up or $browser.gecko or $browser.safari or $browser.opera5up}
             [ <a id="time_tracker_link" class="link" href="javascript:void(null);" onClick="javascript:toggleVisibility('time_tracker');">{get_innerhtml element_name="time_tracker" total=$time_entries|@count}</a> ]
             {/if}
           </td>
--- ./templates/en/view_form.tpl.html~	2005-03-14 16:10:41.000000000 +0200
--- ./templates/en/view_form.tpl.html	2005-03-14 17:04:06.853248450 +0200
@@ -417,7 +417,7 @@
         <tr>
           <td align="left" valign="top" bgcolor="{$cell_color}" width="150">
             <center class="default_white">
-            {if $browser.ie5up or $browser.ns6up or $browser.gecko or $browser.safari}
+            {if $browser.ie5up or $browser.ns6up or $browser.gecko or $browser.safari or $browser.opera5up}
             [ <a id="description_link" class="white_link" href="javascript:void(null);" onClick="javascript:toggleVisibility('description');collapseDescription();">{get_innerhtml element_name="description"}</a> ]
             {/if}
             </center>
--- ./templates/en/checkins.tpl.html~	2005-03-14 17:15:20.000000000 +0200
+++ ./templates/en/checkins.tpl.html	2005-03-14 17:20:01.690035043 +0200
@@ -32,7 +32,7 @@
             <b>SCM Integration - Checkins</b>
           </td>
           <td align="right" class="default">
-            {if $browser.ie5up or $browser.ns6up or $browser.gecko}
+            {if $browser.ie5up or $browser.ns6up or $browser.gecko or $browser.safari or $browser.opera5up}
             [ <a id="checkins_link" class="link" href="javascript:void(null);" onClick="javascript:toggleVisibility('checkins');">{get_innerhtml element_name="checkins" total=$checkins|@count}</a> ]
             {/if}
           </td>
--- ./templates/en/custom_fields.tpl.html~	2005-03-14 17:15:20.000000000 +0200
+++ ./templates/en/custom_fields.tpl.html	2005-03-14 17:23:28.547977094 +0200
@@ -22,7 +22,7 @@
             <b>Issue Details</b>
           </td>
           <td align="right" class="default">
-            {if $browser.ie5up or $browser.ns6up or $browser.gecko}
+            {if $browser.ie5up or $browser.ns6up or $browser.gecko or $browser.safari or $browser.opera5up}
             [ <a id="custom_fields_link" class="link" href="javascript:void(null);" onClick="javascript:toggleVisibility('custom_fields');">{get_innerhtml element_name="custom_fields"}</a> ]
             {/if}
           </td>
--- ./templates/en/attachments.tpl.html~	2005-03-14 17:15:20.000000000 +0200
+++ ./templates/en/attachments.tpl.html	2005-03-14 17:24:41.757925121 +0200
@@ -41,7 +41,7 @@
             <b>Attached Files ({$files|@count})</b>
           </td>
           <td align="right" class="default">
-            {if $browser.ie5up or $browser.ns6up or $browser.gecko}
+            {if $browser.ie5up or $browser.ns6up or $browser.gecko or $browser.safari or $browser.opera5up}
             [ <a id="attachments_link" class="link" href="javascript:void(null);" onClick="javascript:toggleVisibility('attachments');">{get_innerhtml element_name="attachments" total=$files|@count}</a> ]
             {/if}
           </td>
--- ./templates/en/impact_analysis.tpl.html~	2005-03-14 17:15:20.000000000 +0200
+++ ./templates/en/impact_analysis.tpl.html	2005-03-14 17:25:22.950143973 +0200
@@ -40,7 +40,7 @@
             <b>Impact Analysis</b>
           </td>
           <td align="right" class="default">
-            {if $browser.ie5up or $browser.ns6up or $browser.gecko}
+            {if $browser.ie5up or $browser.ns6up or $browser.gecko or $browser.safari or $browser.opera5up}
             [ <a id="impact_analysis_link" class="link" href="javascript:void(null);" onClick="javascript:toggleVisibility('impact_analysis');">{get_innerhtml element_name="impact_analysis"}</a> ]
             {/if}
           </td>
--- ./templates/en/list.tpl.html~	2005-03-14 17:15:20.000000000 +0200
+++ ./templates/en/list.tpl.html	2005-03-14 17:25:47.874040954 +0200
@@ -111,7 +111,7 @@
           </td>
           {if $current_role != $roles.customer}
           <td colspan="2" align="right" class="default">
-            {if $browser.ie5up or $browser.ns6up or $browser.gecko}
+            {if $browser.ie5up or $browser.ns6up or $browser.gecko or $browser.safari or $browser.opera5up}
             hide/show:
             [ <a class="link" title="hide / show the quick search form" href="javascript:void(null);" onClick="javascript:toggleVisibility('filter_form');">quick search</a> ]
             {if $current_role > $roles.reporter}[ <a class="link" title="hide / show the advanced search form" href="javascript:void(null);" onClick="javascript:toggleVisibility('custom_filter_form');">advanced search</a> ]{/if}
--- ./js/dynCalendar.js~	2005-03-14 17:28:14.000000000 +0200
+++ ./js/dynCalendar.js	2005-03-14 17:51:26.879651921 +0200
@@ -220,7 +220,7 @@
 */
 	function dynCalendar_writeHTML()
 	{
-		if (is_ie5up || is_nav6up || is_gecko) {
+		if (is_ie5up || is_nav6up || is_gecko || is_opera5up) {
 			document.write(sprintf('<a href="javascript: %s.show()"><img src="%sdynCalendar.gif" border="0" width="16" height="16" /></a>', this.objName, this.imagesPath));
 			document.write(sprintf('<div class="dynCalendar" id="%s" onmouseover="%s._mouseover(true)" onmouseout="%s._mouseover(false)"></div>', this.layerID, this.objName, this.objName));
 		}
